### PR TITLE
Fixes #2209 by adding collapsed to className

### LINKED ISSUE #2209: Accordion Panel .collapsed class missing
Just updated to 0.30.3 but it seems fixes #2133 and #2137 are not effective.
Anchor toggles between true and false values for aria-expanded and aria-selected, but no collapsed class appears.

### PR DESCRIPTION
**Pull request by ghusse**
Adds `collapsed` in the className following the example in bootstrap docs.

### CODE REVIEW

File: src/Panel.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ class Panel extends React.Component {
         role={role}
         href={id && `#${id}`}
         onClick={this.handleClickTitle}
+        className={ expanded ? '' : 'collapsed' }
```
Comment (jquense) [inline src/Panel.js]: can we use `classNames(expanded && 'collapsed')`? it'll prevent having an empty `class` attribute.

Comment (ghusse) [inline src/Panel.js]: I just wanted to follow the docs.

Is it an issue to have an empty className? Why?

Comment (jquense) [inline src/Panel.js]: its just unnecessary if avoidable, cleaner markup.

**Comment by ghusse** [comment]
I changed the way className is added, by only adding it when necessary.

File: src/Panel.js [hunk 2]
```diff
@@ -94,15 +94,21 @@ class Panel extends React.Component {
...
+      role,
+      href: id && `#${id}`,
+      onClick: this.handleClickTitle,
+      'aria-controls': id,
+      'aria-expanded': expanded,
+      'aria-selected': expanded,
+    };
+
+    if (!expanded) {
+      props.className = 'collapsed';
```
Comment (taion) [inline src/Panel.js]: you don't need to do all this, just keep the previous code but do `className={expanded ? null : 'collapsed'}` if you really don't want to use `classNames` there.

File: test/PanelSpec.js [hunk 3]
```diff
@@ -284,6 +284,7 @@ describe('<Panel>', () => {
       );
       const anchor = ReactDOM.findDOMNode(instance).querySelector('.panel-title a');
       assert.equal(anchor.getAttribute('aria-expanded'), 'true');
+      assert.equal(anchor.getAttribute('class'), null);
```
Comment (taion) [inline test/PanelSpec.js]: just check `anchor.className` as elsewhere

**Comment by ghusse** [comment]
Closed in favor of #2276

**Pull request closed without merging**